lib/plugins/helper/toc.js

```diff
@@ -1,35 +1,29 @@
 'use strict';
 
-let cheerio;
-const { escapeHTML } = require('hexo-util');
+const { tocObj } = require('hexo-util');
 
 function tocHelper(str, options = {}) {
-  if (!cheerio) cheerio = require('cheerio');
+  options = Object.assign({
+    max_depth: 6,
+    class: 'toc',
+    list_number: true
+  }, options);
```

Comment:
`min_depth` option. I don't mind if you prefer to add that option in a separate PR.

Reply:
Maybe in this PR then since `tocObj` already has `min_depth` bring up.

Reply:
@curbengh No. The current unit test for toc helper is completely a mess. I might refactor the unit test when bringing up the `min_depth`.